components/bin/src/bin/stmgr.rs

```diff
@@ -43,26 +43,124 @@ enum InitNodeDbError {
     FileDbError,
 }
 
-fn init_node_db(matches: &ArgMatches) -> Result<(), InitNodeDbError> {
-    let idfile = matches.value_of("idfile").unwrap();
-    let output = matches.value_of("output").unwrap();
-    // Make sure that output_path does not exist.
-    // This program should never override any file! (Otherwise users might erase their database by
+#[derive(Debug, StructOpt)]
+struct InitNodeDbCmd {
+    /// StCtrl app identity file path
+    #[structopt(parse(from_os_str), short = "i", long = "idfile")]
+    idfile: PathBuf,
+    /// Database output file path
+    #[structopt(parse(from_os_str), short = "o", long = "output")]
+    output: PathBuf,
+}
+
+#[derive(Debug, StructOpt)]
+struct GenIdentCmd {
+    /// Identity file output file path
+    #[structopt(parse(from_os_str), short = "o", long = "output")]
+    output: PathBuf,
+}
+
+#[derive(Debug, StructOpt)]
+struct AppTicketCmd {
+    /// StCtrl app identity file path
+    #[structopt(parse(from_os_str), short = "i", long = "idfile")]
+    idfile: PathBuf,
+    /// Application ticket output file path
+    #[structopt(parse(from_os_str), short = "o", long = "output")]
+    output: PathBuf,
+    /// Permission to request reports
```

Comment:
Comment should be: "Permission to request routes"

Reply:
Could you change it? This PR is already merged, I don't think it's worth it for me to make another one.

Reply:
Done on PR: https://github.com/freedomlayer/offst/pull/176